driver-core/src/main/java/com/datastax/driver/core/OpenTelemetryConfiguration.java

```diff
@@ -0,0 +1,60 @@
+package com.datastax.driver.core;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.exporter.zipkin.ZipkinSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+
+public interface OpenTelemetryConfiguration {
```

Comment:
This should be moved to examples module - I would probably even inline it to `ZipkinConfiguration` file.